serverless/packages/functions/src/user/create_user.ts

```diff
@@ -0,0 +1,61 @@
+import { APIGatewayProxyHandlerV2, APIGatewayProxyEventV2 } from "aws-lambda";
+import parseJwt from "../../../core/src/parseJwt";
```

Comment:
interesting that we have to use the full relative path without the `"@/core/...` thingy

Reply:
I'm not 100% why this is this case, I mentioned it on the notion guide but it appears that Vitest is unable to read the import path when we use the shortcut. (If I switch back to the path, Vitest says that it can't find parseJWT:
<img width="1440" alt="image" src="https://github.com/DSGT-DLP/Deep-Learning-Playground/assets/116283570/bbe38ddd-cfd4-4eeb-b6b3-7abf9a9005df">
It seems the solution is as linked here:
[vite project paths](https://dev.to/tilly/aliasing-in-vite-w-typescript-1lfo)
Basically, we need to give vite the same path alias that we declare in tsconfig.json. However, it doesn't look like we have the vite.config.ts file, and I'm not sure why.

Reply:
Maybe we could make a vite.config.ts to make sure that Vitest can read the import path without all this complicated full relative path @ssh51117

Reply:
I take a look, but it'll take me a couple days since I'm busy right now @karkir0003

Reply:
No worries. Let's give it a try for a couple days. If we cannot resolve this by Thursday, we can do it in a followup PR @ssh51117